dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server

```diff
@@ -1221,6 +1221,15 @@ serverCmd()
   SAVE_OPENJ9_JAVA_OPTIONS=${OPENJ9_JAVA_OPTIONS}
   OPENJ9_JAVA_OPTIONS=${SERVER_IBM_JAVA_OPTIONS}
 
+  # Check for any verbose:gc variable set by user
+  # By default we set this to be true unless user specifies otherwise
+  # if [ ! jvmargs, javaoptions, jvmoptionsquoted contains verbose:gc, verbosegc, etc] then SERVER_IBM_JAVA_OPTIONS += -Xverbosegclog:verbosegc.%seq.log,10,1024
+  if ! [[ $JVM_OPTIONS_QUOTED == *"verbosegc"* || $JVM_OPTIONS_QUOTED == *"verbose:gc"* || "${VERBOSEGC}" == "false" ]]
```

Comment:
Another thing to double check is that if the customer specifies `-Xverbosegclog` in their jvm.options, that their options still take precedence. Alternatively, you could also check for `-Xverbosegclog` and skip adding it if it's already there... which might be cleaner anyway.

Reply:
JVM_OPTIONS_QUOTED contains all the jvm.options specified by the user so `JVM_OPTIONS_QUOTED == *"verbosegc"*` catches that, though it would be cleaner to check specifically for `Xverbosegclog` if theres not another way to activate it that contains `verbosegc`. The precedence is checked in the FAT tests, specifically the testJvmChangeVerbose test.